Formula/eudev.rb

```diff
@@ -0,0 +1,36 @@
+class Eudev < Formula
+  desc "Device file manager for the Linux kernel"
+  homepage "https://wiki.gentoo.org/wiki/Eudev"
+  url "https://dev.gentoo.org/~blueness/eudev/eudev-3.2.10.tar.gz"
+  sha256 "87bb028d470fd1b85169349b44c55d5b733733dc2d50ddf1196e026725ead034"
+  license "GPL-2.0-or-later"
+
+  depends_on :linux
+
+  uses_from_macos "gperf" => :build
+
+  conflicts_with "systemd", because: "both install `udev` (a.k.a. `libudev`)"
```

Comment:
The `conflicts_with` field is for conflicts with other formulae. We don't ship a `systemd` formula, so this line is not needed.